From 809d227d66e87c502baf61eadf9bde6b37a4d3d0 Mon Sep 17 00:00:00 2001
From: Mithil Bavishi <bavishimithil@gmail.com>
Date: Sat, 20 Aug 2022 12:46:54 +0530
Subject: [PATCH] dt-bindings: input: twl-pwrbutton: Add support for
 twl6030-pwrbutton

Adds documentation for the compatible string for twl6030 powerbutton
support.

Signed-off-by: Paul Kocialkowski <contact@paulk.fr>
Signed-off-by: Mithil Bavishi <bavishimithil@gmail.com>
Acked-by: Rob Herring <robh@kernel.org>
---
 Documentation/devicetree/bindings/input/twl4030-pwrbutton.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/input/twl4030-pwrbutton.txt b/Documentation/devicetree/bindings/input/twl4030-pwrbutton.txt
index f5021214edec..9a0b765d3af0 100644
--- a/Documentation/devicetree/bindings/input/twl4030-pwrbutton.txt
+++ b/Documentation/devicetree/bindings/input/twl4030-pwrbutton.txt
@@ -8,6 +8,7 @@ This module provides a simple power button event via an Interrupt.
 Required properties:
 - compatible: should be one of the following
    - "ti,twl4030-pwrbutton": For controllers compatible with twl4030
+   - "ti,twl6030-pwrbutton": For controllers compatible with twl6030
 - interrupts: should be one of the following
    - <8>: For controllers compatible with twl4030
 
-- 
2.30.2

